index.html

```diff
@@ -3,177 +3,11 @@
   <head>
 
         <meta charset="utf-8">
-        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
```

Comment:
Not sure if I absolutely needed to remove the "shrink-to-fit" bit, but I was trying to fix a problem with the layout and removed it at one point.  Haven't tried putting it back in, but it doesn't seem to be necessary.